pkg/sql/parser/sql.y

```diff
@@ -4749,42 +4749,42 @@ logical_replication_options:
 // %Help: CREATE VIRTUAL CLUSTER - create a new virtual cluster
 // %Category: Experimental
 // %Text:
-// CREATE VIRTUAL CLUSTER [ IF NOT EXISTS ] name [ <replication> ]
+// CREATE VIRTUAL CLUSTER [ IF NOT EXISTS ] <virtual_cluster_spec> [ <replication> ]
```

Comment:
if we're not doc'ing this let's leave the usage doc text as 'name'.

Reply:
Done.